app/src/main/res/values/strings.xml

```diff
@@ -188,6 +188,8 @@
     <string name="decrypt_message">Enter the password to continue</string>
     <string name="not_encrypted">PDF is not Encrypted</string>
     <string name="save_current">Save Current</string>
+    <string name="save">Save</string>
+    <string name="choose_color_text">Choose a color :</string>
```

Comment:
Add these strings for other locale files also (no need to translate - add them in english only)